src/stdlib_stats.fypp

```diff
@@ -305,11 +306,65 @@ module stdlib_stats
   end interface mean
 
 
+  interface median
+    !! version: experimental
+    !!
+    !! Median of array elements
+    !! ([Specification](../page/specs/stdlib_stats.html#median-median-of-array-elements))
+    #:for k1, t1, o1 in IR_KINDS_TYPES_OUTPUT
+      #:for rank in RANKS
+        #:set RName = rname("median_all",rank, t1, k1, o1)
+        module function ${RName}$ (x, mask) result(res)
```

Comment:
This is just a nitpick, but since Fortran is case insensitive, I wonder if having consistent "styles" for both the preprocessor and Fortran code would mean avoiding having variables/functions with the same name but different capitalization?

Reply:
I am not sure to understand your comment. Do you suggest to use another name than "RName" to avoid capitalization?

Reply:
Yes. In Fortran code
```
real :: RName
integer :: rname
RName = rname
```
would not be allowed, because Fortran does not recognize capitalization. 

This was merely a stylistic suggestion, that it might be good to follow the same rule in fypp code, and reduce the mental burden of switching between capitalization sensitive and capitalization insensitive modes.

Reply:
Would `genname` be fine, instead of `RName`?

Reply:
How about just `name`?

Reply:
Thank you @ivan-pi for the suggestion. It is now renamed `name`.